cantools/database/can/formats/dbc.py

```diff
@@ -420,6 +435,59 @@ def long_to_short_names_dict(self):
         return self._long_to_short_names_dict
 
 
+class UniqueNamesDict(object):
```

Comment:
Do you have plans on affing functionality to this class? If not I suggest replacing it with a function.

Reply:
No more plans, so I'll convert it into a function.